crypto/x509v3/pcy_data.c

```diff
@@ -40,11 +40,11 @@ X509_POLICY_DATA *policy_data_new(POLICYINFO *policy,
 {
     X509_POLICY_DATA *ret;
     ASN1_OBJECT *id;
-    if (!policy && !cid)
+    if ((policy == NULL) && (cid == NULL))
```

Comment:
The extra paren's are harmless.  Some team members like them, some don't (I don't).  But that doesn't hold this back.  :)